[DOCS-6124] 365 Connector

oliveralfresco2018 opened the pull request:
Just sending this over to you to see if you can see this PR ?

gravitonian reviewed (changes requested):
Found microsoft-365/latest/ in two places in microsoft-365/1.0.
Shouldn't microsoft-365/latest/support/index.md include support for ACS 7?

oliveralfresco2018 commented:
> Found microsoft-365/latest/ in two places in microsoft-365/1.0. 
     * Updated cheers.
> Shouldn't microsoft-365/latest/support/index.md include support for ACS 7?
     * Added support for 7.0 cheers.
